lib/src/utils/extensions.dart

```diff
@@ -97,6 +97,8 @@ extension StringExtensions on String? {
     final List<String> nameParts = this!.split(RegExp(r'\W+'))..removeWhere((item) => item.isEmpty);
     if (nameParts.isEmpty) return '';
     return (nameParts.length > 1
+            // TODODE: Should the initials be the first letter of the first name and the last name or the second name?
+            // ? nameParts[0].substring(0, 1) + nameParts[nameParts.length - 1].substring(0, 1)
```

Comment:
I _guess_ it should be the last name?
Perhaps we need to reconsider this function entirely?

Reply:
I've decided to change it to the first and last names rather than the first and second names as I think in it's current state that is what would be expected